docs: add missing time window for social team nomination acceptance (#504)
nodejs/admin: social-team.md

### Nominations

> Nominations and self-nominations are open to **project members** on an ongoing basis. An issue should be created in [nodejs/admin](https://github.com/nodejs/admin) for any kind of nomination. After seven days, if a nomination has not been objected to, it is considered accepted.

Reviewer: This is fine, but I definitely prefer the less-passive and less-tentative wording from the Moderation Policy:

```suggestion
Nominations and self-nominations are open to **project members** on an ongoing basis. An issue should be created in [nodejs/admin](https://github.com/nodejs/admin) for any kind of nomination. If there are no objections after seven days, the nomination is automatically accepted.
```

* "If there are no objections after seven days" is more clear, concise, and idiomatic than "After seven days, if a nomination has not been objected to....".
* "...the nomination is automatically accepted" is more clear, concise, and definitive than "it is considered accepted." (What's the difference between "accepted" and "considered accepted" and why should the reader care about that difference, if there even is one?)

Reviewer 2: +1
